noxfile.py

```diff
@@ -606,7 +606,13 @@ def _get_coverage_env() -> Dict[str, str]:
 
     for line in output.strip().splitlines():
         (key, value) = line.split("=", maxsplit=1)
-        env[key] = value.strip('"')
+        # Strip single or double quotes from the variable value
+        # - quote used by llvm-cov differs between Windows and Linux
+        if value and value[0] == "'":
```

Comment:
This looks overly complex because the two branches are the same?

Reply:
Meaning

```python
if value and value[0] in ["'", '"']:
  value = value[1:-1]
```

should also work?

Reply:
Good point, fixed in force-push.